DESCRIPTION

```diff
@@ -0,0 +1,30 @@
+Package: vizlab
```

Comment:
The individual visualizations should be Makefile projects, not packages, this was copied by accident possibly?